resin/README.md

```diff
@@ -0,0 +1,47 @@
+# Agent Check: Resin
+
+## Overview
+
+This check monitors [Resin][1] through the Datadog Agent.
+
+## Setup
+
+### Installation
+
+The Resin check is not included in the [Datadog Agent][2] package, so you will
+need to install it yourself.
+
+### Configuration
```

Comment:
I'm reading that you need to add `<server-default><jvm-arg>-Dcom.sun.management.jmxremote</jvm-arg></server-default>` in `resin.xml` to the clusters you want to monitor. Can you add a link to the proper documentation here?

Reply:
That might be true.  We currently use this with the APM agent which has the embedded JMXfetch that connects locally.

Reply:
Yes, that's what it says you need to connect locally. To connect remotely it needs a different option.

Reply:
I can confirm that you do not need that option if you are connecting via APM.  These were the options that we currently use without any jmx options and we still receive metrics. 

```
<jvm-arg>-Ddd.jmxfetch.enabled=true</jvm-arg>
<jvm-arg>-Ddd.jmxfetch.metrics-configs=datadog/jmx.resin.yaml</jvm-arg>
<jvm-arg>-Ddd.jmxfetch.refresh-beans-period=120</jvm-arg>
```